Ntupler/src/FillerElectron.cc

```diff
@@ -180,9 +180,6 @@ void FillerElectron::fill(TClonesArray *array,
     pElectron->neuHadIso = itEle->pfIsolationVariables().sumNeutralHadronEt;
     pElectron->puIso     = itEle->pfIsolationVariables().sumPUPt;
 
-    pElectron->ecalPFClusIso = (*hEcalPFClusterIsoMap)[eleBaseRef];
```

Comment:
If these need to be restored, let me know. This only impacts AOD, since MINIAOD values are accessed in a different way.